.github/workflows/actionlint.yml

```diff
@@ -1,5 +1,7 @@
 name: Lint GitHub Actions workflows
-on: [ push, pull_request ]
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
How it will work for detect push to master (no PR) and validation after merge to master?

Reply:
```yaml
on:
  push:
    branches:
      - master
  pull_request:
    branches: '*'
```

Reply:
@nrmancuso , ∆∆

Reply:
Done, additionally identified a couple of others that needed the same config.